sync/class.jetpack-sync-module-comments.php

```diff
@@ -40,6 +43,23 @@ public function init_listeners( $callable ) {
 		$this->init_meta_whitelist_handler( 'comment', array( $this, 'filter_meta' ) );
 	}
 
+	/**
+	 * @param array $data       The new, processed comment data.
+	 * @param array $comment    The old, unslashed comment data.
+	 */
+	public function handle_comment_contents_modification( $data, $comment ) {
+
+		if ( $data['comment_content'] != $comment['comment_content'] ) {
+			$action_args = array(
+				$data['comment_ID'],
+				$data,
+				$comment['comment_content'],
+			);
+			do_action( 'modified_comment_contents', $action_args );
```

Comment:
Done! prefixed with 'jetpack' and passed before and after for content, author, author_email, and author_url if they changed.